package.json

```diff
@@ -1,61 +1,58 @@
+  "name": "@smg-automotive/eslint-config",
   "version": "0.0.0-development",
   "description": "A boilerplate package setup",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "files": [
-    "dist/**/*"
-  ],
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
     "version": "npm run build",
-    "build": "rimraf dist && rollup -c",
-    "lint": "eslint --ext ts,js,tsx,jsx,json .",
+    "build": "echo Success",
+    "lint": "eslint --ext ts,js,tsx,jsx,json ./src --ignore-path ./src/__tests__/.eslintignore",
     "format": "npm run lint -- --fix",
-    "typecheck": "tsc --noEmit",
-    "test": "jest",
-    "test:debug": "node --inspect-brk --inspect=127.0.0.1:9229 ./node_modules/jest/bin/jest.js --runInBand"
+    "test": "node src/__tests__/test.js"
   },
   "author": "SMG Automotive",
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/smg-automotive/fe-example-pkg.git"
+    "url": "https://github.com/smg-automotive/eslint-config-pkg.git"
   },
-  "homepage": "https://github.com/smg-automotive/fe-example-pkg#readme",
-  "devDependencies": {
-    "@rollup/plugin-commonjs": "22.0.0",
-    "@rollup/plugin-node-resolve": "13.3.0",
-    "@rollup/plugin-typescript": "8.3.2",
-    "@testing-library/jest-dom": "5.16.4",
-    "@testing-library/react": "13.2.0",
-    "@testing-library/user-event": "14.1.1",
-    "@types/jest": "27.5.0",
+  "homepage": "https://github.com/smg-automotive/eslint-config-pkg#readme",
+  "dependencies": {
+    "@next/eslint-plugin-next": "12.1.6",
+    "@rushstack/eslint-patch": "1.1.3",
     "@typescript-eslint/eslint-plugin": "5.22.0",
+    "@typescript-eslint/eslint-plugin-tslint": "5.22.0",
     "@typescript-eslint/parser": "5.22.0",
     "eslint": "8.14.0",
-    "eslint-config-airbnb": "19.0.4",
+    "eslint-config-next": "12.1.6",
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-jest": "26.1.5",
-    "eslint-plugin-jsx-a11y": "6.5.1",
+    "eslint-plugin-jsdoc": "39.2.9",
     "eslint-plugin-prettier": "4.0.0",
     "eslint-plugin-react": "7.29.4",
     "eslint-plugin-react-hooks": "4.5.0",
-    "eslint-plugin-testing-library": "5.3.1",
+    "eslint-plugin-testing-library": "5.3.1"
+  },
+  "devDependencies": {
+    "chalk": "4.1.2",
     "jest": "28.0.3",
-    "jest-environment-jsdom": "28.0.2",
     "prettier": "2.6.2",
+    "prop-types": "15.8.1",
+    "react": "18.1.0",
+    "react-dom": "18.1.0",
     "rimraf": "3.0.2",
-    "rollup": "2.72.0",
-    "rollup-plugin-dts": "4.2.1",
-    "rollup-plugin-peer-deps-external": "2.2.4",
-    "semantic-release": "19.0.2",
-    "ts-jest": "28.0.1",
-    "ts-node": "10.7.0",
-    "typescript": "4.6.4"
-  }
+    "semantic-release": "19.0.2"
+  },
+  "exports": {
+    "./prettier": "./src/prettier.js",
+    "./next": "./src/next.js",
+    "./react": "./src/react.js",
+    ".": "./src/index.js"
+  },
+  "files": [
+    "src/**/*"
+  ]
```

Comment:
Nitpicking alert.
Can we have these grouped with all package related config at the beginning of the package.json?